lib/task1/dimensions.dart

```diff
@@ -0,0 +1,9 @@
+import 'dart:html';
+
+import 'package:flutter/cupertino.dart';
+
+abstract class Dimensions {
+  static const double rowHeight = 50;
+  static const double cardWidth = 50;
+  static const EdgeInsets defaultInsets = EdgeInsets.all(10);
```

Comment:
Why I can't reuse the same default insetns?

Reply:
You can but its totally unusable. Are you going to have EdgeInsets.only(top: 10); bottom, left, right and so on?

Reply:
As for now I refer the value in two places, I think it is quite useful for the case.
I see your point, an I will reuse the value if this will be useful in the future.